### PR TITLE
Chore change golem messages from v2.12.0 to v2.13.0

### PR DESCRIPTION
**Pull request by rwrzesien**
No issue for this.

### CODE REVIEW

File: concent_api/core/tests/test_core_views.py [hunk 1]
```diff
@@ -480,7 +480,7 @@ def test_send_task_to_compute_without_public_key_should_return_http_400(self):
 
             self._test_400_response(
                 response,
-                error_code=ErrorCode.MESSAGE_VALUE_WRONG_TYPE
+                error_message='Error in Golem Message.'
```
Comment (dybi) [inline concent_api/core/tests/test_core_views.py]: Please make it a constant, i.e. `ERROR_IN_GOLEM_MESSAGE`

File: concent_api/core/tests/test_core_views.py [hunk 2]
```diff
@@ -503,7 +503,7 @@ def test_send_task_to_compute_with_public_key_with_wrong_length_should_return_ht
 
             self._test_400_response(
                 response,
-                error_code=ErrorCode.MESSAGE_VALUE_WRONG_LENGTH
+                error_message='Error in Golem Message.'
```
Comment (dybi) [inline concent_api/core/tests/test_core_views.py]: same here

File: concent_api/core/tests/test_integration_force_payment.py [hunk 3]
```diff
@@ -497,59 +498,6 @@ def test_provider_send_force_payment_with_empty_subtask_results_accepted_list_co
         self._test_400_response(response)
         self._assert_stored_message_counter_not_increased()
 
-    def test_provider_send_force_payment_with_empty_requestor_ethereum_public_key_concent_should_refuse(self):
```
Comment (dybi) [inline concent_api/core/tests/test_integration_force_payment.py]: why was this removed?

File: concent_api/core/tests/test_integration_report_computed_task.py [hunk 4]
```diff
@@ -1645,7 +1645,7 @@ def test_provider_forces_computed_task_report_missing_key_returns_400_error(self
             )
         self._test_400_response(
             response_1,
-            error_code=ErrorCode.MESSAGE_VALUE_WRONG_TYPE,
+            error_message='Error in Golem Message.',
```
Comment (dybi) [inline concent_api/core/tests/test_integration_report_computed_task.py]: please make it a constant

File: concent_api/core/tests/utils.py [hunk 5]
```diff
@@ -163,6 +163,13 @@ def _sign_message(self, golem_message, client_private_key = None):
             self.REQUESTOR_PRIVATE_KEY if client_private_key is None else client_private_key,
         )
 
+    def _generate_ethereum_signature(self, task_to_compute, requestor_ethereum_private_key=None):
```
Comment (dybi) [inline concent_api/core/tests/utils.py]: I see that sometimes ethereum signature is created "manually" (by call to `task_to_compute.generate_ethsig(REQUESTOR_PRIV_ETH_KEY)`) and sometimes by the function above. Why not having it unified?

Comment (pawelkisielewicz) [inline concent_api/core/tests/utils.py]: It is not described in https://github.com/Code-Poets/coding-style/issues/7, if type signatures should be also in testing utils. If it is no problem please add typing to this method. In this case types are pretty obvious, but lets keep it unified (as below :) )

File: concent_api/core/tests/test_integration_force_payment.py (hunk 3 above)
Comment (rwrzesien) [reply concent_api/core/tests/test_integration_force_payment.py]: It is not possible to correctly create `TaskToCompute` with empty `requestor_ethereum_public_key` as its correctness now gets validated during deserialization against `eth_sig`.

File: concent_api/core/tests/utils.py (hunk 5 above)
Comment (rwrzesien) [reply concent_api/core/tests/utils.py]: In most cases it is automatically done in `_get_deserialized_task_to_compute`. But in few auth tests we change something in returned `TaskToCompute`, so signature has to be regenerated manually.

Comment (rwrzesien) [reply concent_api/core/tests/utils.py]: :+1: good point, done.

File: concent_api/core/tests/test_core_views.py (hunk 1 above)
Comment (rwrzesien) [reply concent_api/core/tests/test_core_views.py]: :+1: done

**Comment by cameel** [comment]
@rwrzesien Why has this been closed without a comment?

Has this actually been merged?

**Comment by rwrzesien** [comment]
@cameel Yes, it looks like github handled it itself (maybe because lack of relates issue?), I don't remember closing it manually.
